rcl/test/rcl/test_wait.cpp

```diff
@@ -179,11 +181,11 @@ TEST(CLASSNAME(WaitSetTestFixture, RMW_IMPLEMENTATION), multi_wait_set_threaded)
             // printf("%s\n", ss.str().c_str());
           }
           ret = rcl_wait(&test_set.wait_set, wait_period);
+          ASSERT_NE(ret, RCL_RET_ERROR);
           if (ret != RCL_RET_TIMEOUT) {
             {
               std::stringstream ss;
-              ss << "[thread " << test_set.thread_id << "] Wakeup  (try #" << wake_try_count << ")";
-              // printf("%s\n", ss.str().c_str());
+              ss << "[thread " << test_set.thread_id << "] Wakeup (try #" << wake_try_count << ")";
```

Comment:
I removed most of the printfs except for the one that occurs during timeout (since it's useful as a warning in case the test fails). I noticed that adding or removing printfs "artificially" improved the performance of this test when it was flaky due to the timing overhead of printf.